app/assets/javascripts/application.js

```diff
@@ -41,15 +45,31 @@
     return link.pathname
   }
 
+  GOVUK.feedback.appendHiddenInputs = function (form) {
```

Comment:
It's moments like this when you realise why jQ can be useful, that's so many variables!

Reply:
Yup, the DOM API is pretty brutal in it's verbosity 🤮